src/styles/variables.ts

```diff
@@ -63,7 +63,7 @@ export default {
     fontSizeXLarge: 22,
     fontSizeXXLarge: 28,
     fontSizeXXXLarge: 32,
-    fontSizeNormalHeight: getValueUsingPixelRatio(20, 28),
+    fontSizeNormalHeight: getValueUsingPixelRatio(21, 28),
```

Comment:
What exactly does this change do? Why is it needed for this particular PR?

Reply:
It is connected to emojis being cut off due to having size bigger than the available space. We would have to cc @j-piasecki but I think it is connected to some changes in `Text` component on new architecture.

Reply:
Is this the right PR to be fixing that though? It seems a bit out of place to me, but curious for your thoughts.

Reply:
It would probably not be the one, but this PR enables new architecture, and it is necessary to make the problem become visible 😅

Reply:
I don't think I am quite following. Can you show me the before & after of this change?

Reply:
It has been reported here: https://github.com/Expensify/App/pull/13767#issuecomment-1924470912 and you can see it on the screenshot there. After this change, the emojis are displayed correctly.